app/components/TrackComponent/tests/index.test.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { renderWithIntl } from '@utils/testUtils';
+import { fireEvent } from '@testing-library/dom';
+import TrackComponent from '../index';
+
+describe('<TrackComponent />', () => {
+  it('should render and match the snapshot', () => {
+    const { baseElement } = renderWithIntl(<TrackComponent />);
+    expect(baseElement).toMatchSnapshot();
+  });
+});
+
+it('should contain one TrackComponent', () => {
+  const { getAllByTestId } = renderWithIntl(<TrackComponent />);
+  expect(getAllByTestId('TrackCard').length).toBe(1);
+});
+
+it('should render the track details inside the Card', () => {
+  const trackName = 'See you Again';
+  const artistName = 'Charlie Puth';
+  const { getByTestId } = renderWithIntl(<TrackComponent trackName={trackName} artistName={artistName} />);
+  expect(getByTestId('name')).toHaveTextContent(trackName);
```

Comment:
let's check the alternate as well. You use an If condition to render the trackName/artistName only when it's available and a placeholder otherwise. Write tests to check whether the placeholder is being rendered